### ISSUE TITLE
Parts are in Utility instead of Life Support

### ISSUE THREAD

**Issue by Superbest** [taraniselsu/TacLifeSupport#88]
I am on KSP 1.0.5 and TACLS 0.11.2 which was installed through CKAN (v1.16.1-0-g2e91715).

It seems that I have a "Life Support" category in VAB (great idea by the way), but it is empty. The LS parts instead clutter up "Utility" and make the actual utilities hard to find. I checked the files inside `ThunderAerospace` and indeed they had `Category` set to `Utility`.

What's up with this? I kind of binge-installed a pile of mods, so it's possible that it's related to that... How should I go about diagnosing the problem?

**Comment by fmatosqg** [comment]
Hmmm never noticed that category, should be some particular MOD you're using. Do you have any mods that produce food.

So far I've played a lot with MKS (stopped updating some months ago)  and now on a career in RO with a bunch of mods "bundled", both with tac, so I'd rule them out. 

Wanna list your mods to pinpoint the culprit?

**Comment by Superbest** [comment]
Ah, definitely a mod then. I suspect the planetary base one. Here's a list from the AVC:

```
KSP: 1.0.5 (Win32) - Unity: 4.6.4f1 - OS: Windows 7 Service Pack 1 (6.1.7601) 64bit
AutoAsparagus - 2.0
SCANsat Mission Pack - 1.0.1
Chatterer - 0.9.7.86
Contract Configurator - 1.9.10
Contract Pack: RemoteTech - 2.0.1
Firespitter - 7.1.5
RasterPropMonitor - 0.24.2
Kerbal Attachment System - 0.5.5
Kerbal Engineer Redux - 1.0.19.4
Kerbal Inventory System - 1.2.5
KSP-AVC Plugin - 1.1.5
Olympic1's Icon Packs - 0.8
Kerbal Planetary Base Systems - 1.0.6
PlanetShine - 0.2.4.2
Protractor Continued - 2.5.1
RCS Build Aid - 0.7.7
RealChute - 1.3.2.7
RemoteTech - 1.6.9
SCANsat - 1.1.4.9
SmartParts - 1.6.6
SmartStage - 2.8
StageRecovery - 1.6.3
TAC Life Support - 0.11.2.1
Kerbal Alarm Clock - 3.5
Alternate Resource Panel - 2.7.4
[x] Science! - 4.14
```

By the way, since it's possible to have this category, what do you think of putting your mod's parts in it? Wouldn't be tidier?

**Comment by fmatosqg** [comment]
They're not mine, but I'd obviously benefit from it. But you'd need to make sure that you either turn this config on if the other plugin is present, or submit a PR that includes such category AND won't conflict with your other mod.

**Comment by cdp1337** [comment]
@Superbest yeah, I have a feeling it's from PlanetaryBases too; I have that mod and found it strange that TACLS isn't in Life Support :p

**Comment by taraniselsu** [comment]
This mod does not add a category -- that is a newish feature in KSP -- so another mod has to be adding it, as you seem to have figured out. Feel free to make an MM patch that will put the parts under that category.

I have not had time to research how the categories work, and I'm not sure how they might have changed in 1.1. They added search capability and part "tags" so you can search not just by name, but by the type of part!

**Comment by macluky** [comment]
Where can I copy that idea? i would like to resolve this issue and don't mind creating the patch, just need to know where to look for inspiration

**Comment by JPLRepo** [comment]
It's already been done... you are in the wrong repro. The latest version has this and has two flavors of it.
I have taken over development of the mod as taraniselsu has moved on in the world.
Here is the latest forum thread: http://forum.kerbalspaceprogram.com/index.php?/topic/146465-122-tac-life-support-v0128-release-5th-jan-2017/
Here is the latest github repro: https://github.com/KSP-RO/TacLifeSupport/releases

**Comment by macluky** [comment]
No I'm running 0.12.8 and see the LS parts in the tab, but also under utility

**Comment by JPLRepo** [comment]
You're still in the wrong repro.
https://github.com/KSP-RO/TacLifeSupport/issues/73
